exercises/isogram/isogram_test.py

```diff
@@ -34,6 +34,9 @@ def test_made_up_name_that_is_an_isogram(self):
     def test_duplicated_character_in_the_middle(self):
         self.assertFalse(is_isogram("accentor"))
 
+    def test_isogram_with_duplicated_non_letter_character_and_duplicate_character(self):
```

Comment:
This is too long for a test name. Could it be shortened?

The build fails due to code style check:
```
./exercises/isogram/isogram_test.py:37:80: E501 line too long (88 > 79 characters)
```